docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,35 @@
 Version history
 ===============
 
+5.3.4 (2017 February 21)
+------------------------
+
+Bug fixes:
+
+* ImageJ
+   - fix for a NullPointerException when exporting images that were not opened via 
+     the Bio-Formats importer, and thus do not have a complete OMEXMLMetadata store.
+
+* Java 1.9
+   - fix compile and runtime errors to enable building with Java 1.9
+
+* ECAT7
+   - update to add support for different versions of ECAT7 files (thanks to Torsten Stöter)
+
+Updated build system:
+
+* updated dependency for `ome-model <https://github.com/ome/ome-model>` in the POM to 
```

Comment:
Sorry, this link needs `_` at the end to render as a hyper link - see https://www.openmicroscopy.org/site/support/bio-formats5.3-staging/about/whats-new.html